### DISCUSSION TITLE
Where can I find Registry Workshop materials?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/registry#92] [Q&A] [answered]
Are there recordings and/or minutes of the Registry Workshop meetings?

**Comment by pds-ops** [comment, marked as answer]
We try our best to capture info/notes/meeting recordings on the Registry wiki: https://github.com/NASA-PDS/registry/wiki

From the first 2 workshops:
https://github.com/NASA-PDS/registry/wiki/Registry-Workshop---06_28_2022
https://github.com/NASA-PDS/registry/wiki/Registry-Workshop-07_26_2022
